packages/data-driver-postgres/src/query/index.test.ts

```diff
@@ -0,0 +1,28 @@
+import type { SqlStatement } from '@directus/data-sql';
+import { randomAlpha, randomInteger } from '@directus/random';
+import { beforeEach, expect, test } from 'vitest';
+import { constructSql } from './index.js';
+
+let sample: {
+	statement: SqlStatement;
+};
+
+beforeEach(() => {
```

Comment:
how come `beforeEach` is used here? 🤔 this would assume that each test would basically work with the same query except for the values. I assume we can do some more tests here in the future